README.md

```diff
@@ -1,3 +1,7 @@
+<p align="center">
+  <img alt="qs" src="./logos/banner_dark.png" width="800" />
```

Comment:
hm, the banner ends up pretty tall here (https://github.com/agreea/qs/tree/add-logo#readme) - could we make something that's a lot shorter so it doesn't push the readme content so far down the fold?